gdc_ng_models/models/release_log.py

```diff
@@ -0,0 +1,26 @@
+from sqlalchemy import Boolean, Column, Enum, Index, Text
+from sqlalchemy.ext.declarative import declarative_base
+
+Base = declarative_base()
+
+
+class ReleaseLog(Base):
+
+    __tablename__ = "release_log"
+    __table_args__ = (
+        Index("release_log_project_id_idx", "program_name", "project_code"),
+    )
+
+    def __repr__(self):
+        return "<ReleaseLog(project_id={}-{}, release_number='{}', open='{}')>".format(
+            self.program_name,
+            self.project_code,
+            self.release_number,
+            self.open,
+        )
+
+    program_name = Column(Text, primary_key=True, nullable=False)
+    project_code = Column(Text, primary_key=True, nullable=False)
+    release_number = Column(Text, nullable=False)
+    data_type = Column(Enum("ssm", "cnv", name='data_type'), nullable=False)
```

Comment:
can you define this on the top level instead? this way extending it will be simpler. shouldn't we have `case` in there too?

Reply:
I can add case but I didn't see it anywhere in the ticket. When I asked the channel the only response said that these were the two enum values.

Reply:
Enums are a bit hard to update at the db level, you have to write custom sql scripts to add new ones I think, just something to remember if we decide to use enums

Reply:
yeah, right, maybe a better approach would be to have a custom validator that validates against enum values 🤷‍♂️